src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,213 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.sql.SQLOutput;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    protected static final String LOCAL_FILE = "data/duke.txt";
+
+    public static void appendToFile(String filePath, String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true); // create a FileWriter in append mode
```

Comment:
leaving comments to help readers enables a better understanding of your code and might even help to hasten the onboarding process. Good job!